services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/taydennys/TaydennysServiceITest.kt

```diff
@@ -686,4 +701,119 @@ class TaydennysServiceITest(
             }
         }
     }
+
+    @Nested
+    inner class Delete {
+        @Test
+        fun `deletes the attachments when deleting a taydennys`() {
+            val taydennys = taydennysFactory.saveWithHakemus { it.withMandatoryFields() }
+            attachmentFactory.save(taydennys = taydennys).withContent()
+            attachmentFactory.save(taydennys = taydennys).withContent()
+            assertThat(attachmentRepository.findByTaydennysId(taydennys.id)).hasSize(2)
+            assertThat(
+                    fileClient.list(
+                        Container.HAKEMUS_LIITTEET,
+                        ApplicationAttachmentContentService.prefix(taydennys.hakemusId),
+                    )
+                )
+                .hasSize(2)
```

Comment:
There are no attachments for other hakemus in these tests, so these can all be simplified:
```suggestion
            assertThat(fileClient.listBlobs(Container.HAKEMUS_LIITTEET)).hasSize(2)
```

Reply:
Thank you, changed them.